snippets/swift/BreezSDKExamples/Sources/ListPayments.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ListPayments.swift
+//
+//
+//  Created by ruben on 13/11/2023.
+//
+
+import Foundation
+import BreezSDK
+
+func ListPayments(sdk: BlockingBreezServices) -> [Payment]? {
+    // ANCHOR: list-payments
+    let payments = try? sdk.listPayments(req: ListPaymentsRequest(filters: [.closedChannel, .sent, .received]))
```

Comment:
In the other language snippets, `filters` is empty or not even set.

Reply:
fixed